test/library/standard/Path/victor-ludorum/testcases_isAbsPath.skipif

```diff
@@ -0,0 +1,5 @@
+# It is supportable in 'Unix' and
```

Comment:
This will do the opposite of what you intend, sadly.  It will skip this test on darwin, linux32 and linux64, but those are the operating systems where it will work.

You want to use this file's current contents for the cygwin test's .skipif.
After that, you'll want to change the skipif for this test to check against cygwin32 and cygwin64.

Have you tried running `start_test $CHPL_HOME/test/library/standard/Path/victor-ludorum/` on your machine?